test/es6/HTMLComments.js

```diff
@@ -0,0 +1,72 @@
+﻿//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+/* NOTE: This file needs to be treated as binary. It contains mixed line endings, including non-standard
+ *       line endings. Most text editors will not handle the file correctly. If you need to edit this
+ *       file, make sure you do a binary compare to ensure the non-standard line endings have not been lost.
+ *
+ *       'LS' refers to Unicode Character 'LINE SEPARATOR' (U+2028)
+ *       'PS' refers to Unicode Character 'PARAGRAPH SEPARATOR' (U+2029)
+ */
+
+/*
+ * Line terminator sequences - standard (11.3 LineTerminator)
+ */
+
+// CRLF
+WScript.Echo("Code before CRLF<-- is reachable");
+--> WScript.Echo("Code after CRLF<-- is unreachable");
+WScript.Echo("CRLF<-- does not affect automatic semicolon insertion")

```

Comment:
Is this a sufficient test to prove that semicolon insertion is not affected?  Is it possible to use a grammar production that relies on [no LineTerminator here] to prove that automatic semicolon insertion is doing the right thing?